modules/ROOT/pages/vm/vm-publish-across-apps.adoc

```diff
@@ -33,24 +30,34 @@ So, you can create a VM configuration in a domain like in this:
 
 </mule-domain>
 ----
-
-Then you place a listener in an application of the same domain:
-
++
+. Place a listener in an application of the same domain:
++
 [source,xml,linenums]
 ----
 <flow name="crossAppListener">
 	<vm:listener queueName="sharedQueue" config-ref="sharedVMConfig" />
 	<logger />
 </flow>
 ----
-
-Then you publish messages from a second application, also of the same domain:
-
++
+. Publish messages from a second application, also of the same domain:
++
 [source,xml,linenums]
 ----
 <flow name="crossAppRequester">
 	<vm:publish queueName="sharedQueue" config-ref="sharedVMConfig" />
 </flow>
 ----
-
-Notice how both the listener and publisher point to the same configuration defined in the domain.
\ No newline at end of file
++
+. Pull messages from the queue after they are published.
```

Comment:
I wouldn't say that this is another step, you either put a listener in the queue, or you consume them via the consume operation. this makes it seem like it would be valid to do both.

Reply:
Please tell me how to get the example to be correct. Would the example be correct if I eliminate the listener code frlom it?